sympy/functions/special/zeta_functions.py

```diff
@@ -326,6 +326,19 @@ def _eval_expand_func(self, **hints):
             return expand_mul(start).subs(u, z)
         return polylog(s, z)
 
+    def _eval_at_reals(self, x, side):
+        # polylog is a branched function with the branch cut of the
+        # principal branch along the interval [1, oo), where it is
+        # continous from below by convention. Reference:
+        # http://functions.wolfram.com/ZetaFunctionsandPolylogarithms/PolyLog/04/05/01/
+        from sympy.functions import Piecewise, gamma, unpolarify
+        s, z = self.args
+        z = unpolarify(z)
+        cond = (z > 1) & (z.diff(x)*side > 0)
```

Comment:
This should probably check if z > 1 raises an exception for complex comparison (assumedly in that case `cond` would be False).

Reply:
I wasn't sure what would be finally needed. That seemed to work for the current tests. Maybe a try block should be set up.